package.json

```diff
@@ -13,7 +13,7 @@
     "react-scripts": "3.2.0",
     "recompose": "^0.30.0"
   },
-  "proxy": "http://localhost:8080",
+  "proxy": "http://localhost:3000",
```

Comment:
Hey, was this giving you trouble? Express needs to run on 8080 since react is running on 3000

Reply:
@sherribooher